metrics/web/lib/common/presentation/injector/widget/injection_container.dart

```diff
@@ -242,8 +245,13 @@ class _InjectionContainerState extends State<InjectionContainer> {
       BrowserNavigationState(window.history),
     );
 
+    _featureConfigNotifier =
+        FeatureConfigNotifier(_fetchFeatureConfigUseCase);
```

Comment:
It looks like we are not injecting this FeatureConfig in the build() and rather creating a new one.